lib/error/error_json.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include <utils/error/error_json.hpp>
+
+#include <everest/logging.hpp>
+
+namespace Everest {
+namespace error {
+
+Error json_to_error(const json& j) {
```

Comment:
This could probably be done with to_json and from_json functions like we do in libocpp:
https://github.com/EVerest/libocpp/blob/497b08c58405e0bcc0b4fb23843b063be1f83f99/lib/ocpp/v16/messages/Authorize.cpp#L20

Reply:
Sounds reasonable :+1: